lib/peppermint_sdk.dart

```diff
@@ -16,3 +16,4 @@ export 'package:flutter_secure_storage/flutter_secure_storage.dart';
 export 'package:lottie/lottie.dart';
 export 'package:flutter_svg/flutter_svg.dart';
 export 'src/widgets/nft_scanner.dart';
+export 'src/widgets/loading.dart';
```

Comment:
Remove loading from here

Reply:
Oh, forgot about it, I already created a new commit below, thank you.